src/export.c

```diff
@@ -0,0 +1,36 @@
+#define R_NO_REMAP
+#include <Rinternals.h>
+
+#include "export.h"
+
+// Compat for R 3.4.0
```

Comment:
If these functions are only for compatibility with R versions prior to R 3.4.0 we should conditionally define them `if defined(R_VERSION) && R_VERSION >= R_Version(3, 3, 0)` and use the R definitions if they are available.